README.md

```diff
@@ -200,9 +200,59 @@ doxygen
 
 # Satori
 
-This repository also contains the Satori client for use with the [Satori Liveops Server](https://heroiclabs.com/satori/).
+Satori is a liveops server for games that powers actionable analytics, A/B testing and remote configuration. Use the Satori .NET Client to coomunicate with Satori from within your .NET game.
 
-It follows the same authentication patterns as Nakama but is used for managing your live game via sending analytics events, updating properties, getting feature flags and experiments, and more.
+Full documentation is online - https://heroiclabs.com/docs/satori/client-libraries/unity/index.html
+
+## Getting Started
+
+Create a client object that accepts the API you were given as a Satori customer.
+
+```csharp
+using Satori;
+
+const string scheme = "https";
+const string host = "127.0.0.1"; // add your host here
+const int port = 443;
+const string apiKey = "apiKey"; // add the api key that was given to you as a Satori customer.
+
+var client = new Client(scheme, host, port, apiKey);
+```
+
+Then authenticate with the server to obtain your session.
+
+
+```csharp
+// Authenticate with the Satori server.
+try
+{
+    session = await client.AuthenticateAsync(id);
+    Debug.Log("Authenticated successfully.");
+}
+catch(ApiResponseException ex)
+{
+    Debug.LogFormat("Error authenticating: {0}", ex.Message);
+}
+```
+
+Using the client you can get any experiments or feature flags, the user belongs to.
+
+```csharp
+var experiments = await client.GetExperimentsAsync(session);
+var flag = await client.GetFlagAsync(session, "FlagName");
+```
+
+You can also send arbitrary events to the server:
+
+```csharp
+
+await client.EventAsync(session, new Event("gameLaunched", DateTime.UtcNow));
+
+```
+
+This is only a subset of the Satori client API, so please see the documentation link listed earlier for the full API.
```

Comment:
I think we should add a link to this page: https://dotnet.docs.heroiclabs.com/html/namespace_satori.html